src/Servers/Kestrel/Core/src/Internal/Infrastructure/PipeWriterHelpers/TimingPipeFlusher.cs

```diff
@@ -9,7 +9,7 @@
 using Microsoft.AspNetCore.Server.Kestrel.Core.Internal.Http;
 using Microsoft.Extensions.Logging;
 
-namespace Microsoft.AspNetCore.Server.Kestrel.Core.Internal.Infrastructure
+namespace Microsoft.AspNetCore.Server.Kestrel.Core.Internal.Infrastructure.PipeWriterHelpers
```

Comment:
nit: Idk if we should have this in the PipeWriterHelpers namespace.

Reply:
The only reason I did this is because Kestrel Core's Infrastructure directory was getting pretty large, particularly with the addition of BufferSegment, and I wanted to namespace to match the directory structure.

I guess TimingPipeFlusher could go back to the Infrastructure directory, but if feels more related to the other stuff in the PipeWriterHelpers folder in my opinion.